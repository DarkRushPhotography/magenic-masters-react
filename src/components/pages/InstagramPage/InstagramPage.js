import React, { Component } from "react";

import PageHeader from "../../@shared/PageHeader/PageHeader";
import Page from "../../@shared/Page/Page";

class InstagramPage extends Component {

  render() {
    return (
      <Page>
        <PageHeader />
        <h1>Instagram</h1>
      </Page>
    );
  }
}

export default InstagramPage;
