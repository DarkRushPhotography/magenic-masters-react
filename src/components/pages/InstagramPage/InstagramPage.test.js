import React from "react";
import { shallow } from "enzyme";

import InstagramPage from "./InstagramPage";

describe("InstagramPage", () => {
  it("should render correctly", () => {
    const component = shallow(<InstagramPage />);

    expect(component).toMatchSnapshot();
  });
});