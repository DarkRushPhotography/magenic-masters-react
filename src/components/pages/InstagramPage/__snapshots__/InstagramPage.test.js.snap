// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`InstagramPage should render correctly 1`] = `
<Page>
  <withRouter(WithStyles(PageHeader)) />
  <h1>
    Instagram
  </h1>
</Page>
`;
